controllers/llamaCPP.cc

```diff
@@ -344,3 +344,16 @@ void llamaCPP::loadModel(
   warmupModel();
   callback(resp);
 }
+
+void inferences::llamaCPP::unloadModel(const HttpRequestPtr &req, std::function<void(const HttpResponsePtr &)> &&callback)
+{
+  Json::Value jsonResp;
+  if (model_loaded) {
+    llama.unloadModel();
```

Comment:
Oh, sorry my mistake. The API endpoint map to the wrong handler:
it should be `METHOD_ADD(llamaCPP::unloadModel, "unloadmodel", Delete);`
instead of `METHOD_ADD(llamaCPP::loadModel, "unloadmodel", Delete);`